doc/guide/guide-bloch.rst

```diff
@@ -421,11 +421,13 @@ Directly Generating an Animation
 
 The code to directly generate an mp4 movie of the Qubit decay is as follows ::
 
+   import matplotlib
```

Comment:
I think this `import matplotlib` is no longer needed now?

Reply:
Thank you!